test/test_steno.py

```diff
@@ -59,3 +59,19 @@ def test_steno(self):
         self.assertEqual(Stroke(['-P', '-P']).rtfcre, '-P')
         self.assertEqual(Stroke(['-P', 'X-']).rtfcre, 'X-P')
         self.assertEqual(Stroke(['#', 'S-', '-T']).rtfcre, '1-9')
+
+    def test_stroke_to_indexes(self):
+        cases = (
+            ('STPH', (1, 2, 4, 6,)),
+            ('#', (0,)),
+            ('-T', (19,)),
+            ('STPHA*T', (1, 2, 4, 6, 8, 10, 19,)),
+            ('WUZ', (5, 12, 22,)),
+            ('#KW*', (0, 3, 5, 10,)),
+            ('#-Z', (0, 22,)),
+            ('12K3W*', (1, 2, 3, 4, 5, 10,)),
+            ('ED', (11, 21,)),
+            ('*ED', (10, 11, 21,))
```

Comment:
We should hit the edge cases, all the edges:

`#, *, S-, R-, A-, -U, -F, -Z`.

Mainly concerned with combos of most of these (using judgment)

I didn't originally write it this way, either, but we should probably put the steno through normalization first, just in case, as that is the expected behavior of the program. That way if the normalization function gets updated, the compare tests will start to fail too!

So something like:

`self.assertEqual(indexes, normalized_stroke_to_indexes(normalize_stroke(stroke)))`